wire/msgtx_test.go

```diff
@@ -102,10 +101,10 @@ func TestTx(t *testing.T) {
 		0xa6, // 65-byte signature
 		0xac, // OP_CHECKSIG
 	}
-	txOut := NewTxOut(txValue, pkScript)
```

Comment:
This change isn't really accurate.  The output amount will almost never match the input amount.  The difference is the fee.  I don't think they should be tied together like this.  It might make sense to do `txValue := txIn.ValueIn - 1000` to simulate a fee and then leave all of these references as `txValue`.